src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/PackagesConfigToPackageReferenceMigrator.cs

```diff
@@ -26,7 +26,6 @@ internal static class PackagesConfigToPackageReferenceMigrator
             NuGetProject nuGetProject,
             IEnumerable<NuGetProjectUpgradeDependencyItem> upgradeDependencyItems,
             IEnumerable<PackageIdentity> notFoundPackages,
```

Comment:
It doesn't seem right to just ignore uninstalling notFoundPackages because that will still not allow it to convert to PR. We should rather just fail upgrade when there are notFoundPackages